taxi/views.py

```diff
@@ -2,14 +2,16 @@
 from django.shortcuts import render
 from django.views import generic
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.urls import reverse_lazy
+from django.views.generic import CreateView, UpdateView, DeleteView, DetailView
```

Comment:
The `CreateView`, `UpdateView`, `DeleteView`, `DetailView` imports are not grouped correctly. According to the checklist, you should group imports using `()` if needed. Please, correct it.